src/components/cohorts/AddCohortForm.js

```diff
@@ -0,0 +1,50 @@
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+import Header from '../Header/Header';
+import client from '../../utils/client';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+
+const AddCohortForm = () => {
+  let navigate = useNavigate()
+  const [cohortFormData, setCohortFormData] = useState({ cohort_name: '' });
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    client
+      .post('/cohort', cohortFormData)
+      .then((res) => {
+        navigate(`../cohorts/${res.data.data.cohort.id}`, { replace: true })
```

Comment:
You can shorten this to make it more readable and not so 'data'tastic by destructuring on the line above
`const data = res.data.data;`
then have 
```navigate(`../cohorts/${data.cohort.id}```